internal/cephfs/snapshot.go

```diff
@@ -61,30 +61,16 @@ func (vo *volumeOptions) createSnapshot(ctx context.Context, snapID, volID volum
 	return nil
 }
 
-func deleteSnapshot(ctx context.Context, volOptions *volumeOptions, cr *util.Credentials, snapID, volID volumeID) error {
-	args := []string{
-		"fs",
-		"subvolume",
-		"snapshot",
-		"rm",
-		volOptions.FsName,
-		string(volID),
-		string(snapID),
-		"--group_name",
-		volOptions.SubvolumeGroup,
-		"-m", volOptions.Monitors,
-		"-c", util.CephConfigPath,
-		"-n", cephEntityClientPrefix + cr.ID,
-		"--keyfile=" + cr.KeyFile,
-		"--force",
+func (vo *volumeOptions) deleteSnapshot(ctx context.Context, snapID, volID volumeID) error {
+	fsa, err := vo.conn.GetFSAdmin()
+	if err != nil {
+		util.ErrorLog(ctx, "could not get FSAdmin, can delete snapshot %s from %s/%s: %s", string(snapID), vo.FsName, string(volID), err)
```

Comment:
```suggestion
util.ErrorLog(ctx, "could not get FSAdmin %s", err)
```
Log only error?

Reply:
sure, will do. I guess we assume the req-id and other details are logged at different locations.